tests/std/tests/VSO_0000000_vector_algorithms_mismatch_and_lex_compare/test.cpp

```diff
@@ -172,9 +172,19 @@ namespace test_mismatch_sizes_and_alignments {
 
     template <class T, size_t Size, size_t PadSize>
     char stack_array_various_alignments_impl() {
+#ifdef __clang__
+#if __has_feature(undefined_behavior_sanitizer)

```

Comment:
Do you prefer nested preprocessor instead of preprocessor logical and here?

Reply:
MSVC treats `defined(__clang__) && __has_feature(undefined_behavior_sanitizer)` as a syntax error, so it's necessary to use nested preprocessor here.